doc/README.md

```diff
@@ -322,6 +323,59 @@ config:
     disco_srv: http://disco.example.com
 ```
 
+##### Mirror the SAML ForceAuthn option
+
+By default when the SAML frontend receives a SAML authentication request
+with `ForceAuthn` set to `True`, this information is not mirrored in the SAML
+authentication request that is generated by the SAML backend towards the
+upstream identity provider. If the configuration option
+`mirror_force_authn` is set to `True`, then the default behaviour changes
+and the SAML backend will set `ForceAuthn` to true when it proxies a SAML
+authentication request with `ForceAuthn` set to `True`.
+
+The default behaviour is `False`.
+
+```yaml
+config:
+  mirror_force_authn: True
+  [...]
+```
+
+##### Memorize the IdP selected through the discovery service
+
+In the classic flow, the user is asked to select their home organization to
+authenticate to. The `memorize_idp` configuration option controls whether
+the user will have to always select a target provider when a discovery service
+is configured. If the parameter is set to `True` (and `ForceAuthn` is not set),
+the proxy will remember and reuse the selected target provider for the duration
+that the state cookie is valid. If `ForceAuthn` is set, then the
+`use_memorized_idp_when_force_authn` configuration option can overide
+this property and still reuse the selected target provider.
+
+The default behaviour is `False`.
+
+```yaml
+config:
+  memorize_idp: True
+  [...]
+```
+
+##### Use the configured discovery service if ForceAuthn is set to true
```

Comment:
Using `use_memorized_idp_when_force_authn` allows backwards compatibility as its absence means it is set to `False`. This was previously the option `use_disco_when_forceauthn` which had the invert logic - it needed the default value `True` which feels non-intuitive.